gravitee-apim-cypress/cypress/fixtures/fakers/applications.ts

```diff
@@ -19,9 +19,14 @@ import { Application } from '@model/applications';
 export class ApplicationFakers {
   static application(attributes?: Partial<Application>): Application {
     return <Application>{
-      ...attributes,
       name: faker.commerce.productName(),
-      description: faker.commerce.productDescription(),
+      description: 'This application was created by a Cypress test',
```

Comment:
why can't we keep the random description ?

Reply:
Yes, we can do so, no problem. I just noticed that if you mix manual tests and automated tests on one installation then it's sometimes quite nice to see where an entity came from. And since there are no problems with entities having the same description I used the chance to put a more meaningful description there. But that's just my thought about it, wdyt?

Reply:
it's ok to me :+1: